afs/media/js/views/components/reports/instrument.js

```diff
@@ -40,63 +36,52 @@ define([
             self.documentationDataConfig = {
                 'subjectOf': undefined, 
             };
-            self.existenceDataConfig = {
-                'production': {
-                    graph: 'production',
-                    metadata: [{
-                        key: 'production event type',
-                        path: 'production_type',
-                        type: 'resource'
-                    },{
-                        key: 'producer',
-                        path: 'production_carried out by',
-                        type: 'resource'
-                    },{
-                        key: 'production event location',
-                        path: 'production_location',
-                        type: 'resource'
-                    }]
-                },
-            };
+            
             self.nameCards = {};
             self.descriptionCards = {}
             self.documentationCards = {};
-            self.existenceCards = {};
-            self.substanceCards = {};
             self.summary = params.summary;
 
+            self.textualReferenceTableConfig = {
+                ...self.defaultTableConfig,
+                columns: Array(3).fill(null)
+            };
+
+            self.visible = {
+                textualReference: ko.observable(true),
+            }
+            
+            self.textualReference = ko.observableArray();
+            self.textualReferenceDisplay = ko.observable(true);
+            
+            const textualReferenceData = self.resource()['Textual Reference'];
+            if(textualReferenceData) {self.textualReference(textualReferenceData.map(x => {
+                const type = x['Textual Reference Type']['@display_value'];
+                const link = reportUtils.getResourceLink({resourceId: x['Textual Source']['resourceId']});
+                const source = x['Textual Source']['@display_value'];
```

Comment:
There's a function to get the @display_value for you in report.js.  See getNodeValue